include/items.php

```diff
@@ -144,24 +144,43 @@ function query_page_info($url, $photo = "", $keywords = false, $keyword_blacklis
 function add_page_keywords($url, $photo = "", $keywords = false, $keyword_blacklist = "")
 {
 	$data = query_page_info($url, $photo, $keywords, $keyword_blacklist);
+	if (empty($data['keywords']) || !is_array($data['keywords'])) {
```

Comment:
Code standards: Please normalize quote style in this function, either all double quotes or all single quotes. Since it's an existing function I'd advocate for double quotes.